### PR TITLE
Pickups speed adjusted. Closes #694

### LINKED ISSUE #694: Pickups don't seem to be rotating fast enough
When visually compared to GTA III on iOS, pickups in OpenRW seem to be rotating slower (the coronas are pulsating correctly, though). Could someone please provide the correct rotating speed multiplier to make pickups spin with the same exact speed as in the original engine?

### PR DESCRIPTION
**Pull request by ArtemPopof**
https://github.com/rwengine/openrw/issues/694
Now rotation speed is almost exactly as in original GTA III on my pc.

### CODE REVIEW

File: rwengine/src/render/ObjectRenderer.cpp [hunk 1]
```diff
@@ -308,9 +308,11 @@ void ObjectRenderer::renderVehicle(VehicleObject* vehicle,
 void ObjectRenderer::renderPickup(PickupObject* pickup, RenderList& outList) {
     if (!pickup->isEnabled()) return;
 
+    static const auto rotationSpeedCoeff = 3;
```
Comment (JorgenPo) [inline rwengine/src/render/ObjectRenderer.cpp]:
```suggestion
    static constexpr float rotationSpeedCoeff = 3.0f;
```

Comment (danhedron) [inline rwengine/src/render/ObjectRenderer.cpp]:
```suggestion
    constexpr auto kRotationSpeedCoeff = 3.f;
```

**Comment by dracc** [comment]
Please consider changing your PR description so that it includes `Closes #694`.
Apart from that, code looks good. I have not tested it myself though.

**Comment by ShFil119** [comment]
Can you extract last commit to another pull?

**Comment by ArtemPopof** [comment]
Yes, my mistake.

**Comment by ArtemPopof** [comment]
Oops

**Comment by ShFil119** [comment]
Please visit IRC, it's much easier to discuss stuff.

**Comment by ShFil119** [comment]
Thanks. :)

**Pull request merged**